app/views/talk/show.html.erb

```diff
@@ -1,4 +1,4 @@
 <div><p class="help-text" style="margin-top: 10px;clear:left;">
-  <%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => @node.path, :url2 => '/wiki/talk-pages') %> | for help, <a href="/chat">visit our chatroom</a>
+  <%= raw t('talk.show.welcome', page: @node.latest.title, url: @node.path, url2: '/wiki/talk-pages') %> | for help, <a href="/chat">visit our chatroom</a>
```

Comment:
Hi @chanjess , there is a typo in this line 2, see `url1`. Thanks so much for helping!